client/app/components/dashboards/gridstack/index.js

```diff
@@ -89,7 +89,8 @@ function gridstack($parse, dashboardGridOptions) {
             item.minSizeX, item.maxSizeX, item.minSizeY, item.maxSizeY,
             itemId,
           );
-          grid._updateStyles(grid.grid.getGridHeight());
+          const gridHeight = grid.grid.getGridHeight() + 10;
```

Comment:
What's the purpose of the magic number (10)? :)

Reply:
Actual fix for the issue is few lines below (`grid._updateStyles` in `finally` section); I added this `10` because `gridstack` also adds it to preserve some extra space.